pkg/controller/migmigration/restore.go

```diff
@@ -222,8 +224,25 @@ func (t *Task) buildRestore(backupName string) (*velero.Restore, error) {
 // Update a Restore as desired for the destination cluster.
 func (t *Task) updateRestore(restore *velero.Restore, backupName string) {
 	restorePVs := true
+
+	namespaceMapping := make(map[string]string)
+	for _, namespace := range t.namespaces() {
+		mapping := strings.Split(namespace, ":")
+		if len(mapping) == 1 {
+			namespaceMapping[namespace] = namespace
+		}
+		if len(mapping) == 2 {
+			if mapping[1] == "" {
+				namespaceMapping[mapping[0]] = mapping[0]
```

Comment:
I get why this check is here, but it feels weird a user could have `test:` as a valid namespace mapping.

Reply:
So based on this code, "test", "test:", and "test:test" are all equivalent. As long as velero interprets the three this way, this seems like the right way to handle it.

Reply:
Actually, looking at the upstream code, I think it would be cleaner to omit namespaces from the NamespaceMapping if they're not being mapped to something else. Looking at this code, an unmapped namespace will just keep its original value:
```
                        // get target namespace to restore into, if different
                        // from source namespace
                        targetNamespace := namespace
                        if target, ok := ctx.restore.Spec.NamespaceMapping[namespace]; ok {
                                targetNamespace = target
                        }
```